python-formatted-output/interpolation.py

```diff
@@ -0,0 +1,46 @@
+# print(f"Hello, Pythonista!")
+# print(f"Hello, Pythonista!")
+
+# print(f"Single-line f-string with single quotes")
+# print(f"Single-line f-string with double quotes")
+# print(
+#     f"""Multiline triple-quoted f-string
+# with single quotes"""
+# )
+# print(
+#     f"""Multiline triple-quoted f-string
+# with double quotes"""
+# )
```

Comment:
I think Black has already messed up these examples (before the comments where added?). So we should redo the f vs F and single vs double quotes.